tests/api/test_auth.py

```diff
@@ -293,3 +293,41 @@ def test_get_user_from_macaroon_expired(self):
             yield self.auth.get_user_from_macaroon(macaroon.serialize())
         self.assertEqual(401, cm.exception.code)
         self.assertIn("Invalid macaroon", cm.exception.msg)
+
+    @defer.inlineCallbacks
+    def test_get_user_from_expired_access_token(self):
+        self.get_user_from_macaroon = Mock(return_value=AuthError)
+        self.hs.config.expire_access_token = True
+        with self.assertRaises(AuthError) as cm:
+            yield self.auth.get_user_by_access_token('alsdghsdfvedshoeng')
+        self.assertEqual(401, cm.exception.code)
+        self.assertIn("access token has expired", cm.exception.msg)
```

Comment:
Sure I will remove this actually forgot to do it, wanted to know if the exception raises for invalid token and for my own assurance :)